third_party/terraform/website/docs/r/container_cluster.html.markdown

```diff
@@ -350,6 +350,11 @@ maintenance_policy {
 
 The `ip_allocation_policy` block supports:
 
+* `use_ip_aliases` - (Optional) Whether alias IPs will be used for pod IPs in
+the cluster. Defaults to `true` if the `ip_allocation_policy` block is defined,
+and to the API default otherwise. Prior to March 31, 2019, the default on the
```

Comment:
According to emails received today, this changed was postponed to June 17th:

> From: Google Cloud Platform <cloud-proactive-comms+****@google.com> 
> Sent: Monday, March 4, 2019 15:32 
> Subject: [Reminder] GKE cluster creation default mode will change to VPC-Native from June 17
> 
> Hello Google Kubernetes Engine Customer, 
> 
> We previously sent you an email stating that from March 31, 2019, newly created clusters will, by default, be VPC-native instead of routes-based. We are writing to let you know that this change will now take place from June 17, 2019. 
[...]

Reply:
Thanks! For whatever reason I'm not on the list. I'll update the message.